prog/download_boot_image.rb

```diff
@@ -41,26 +37,7 @@ def default_boot_image_version(image_name)
   end
 
   def url
-    @url ||=
-      if frame["custom_url"]
-        frame["custom_url"]
-      elsif download_from_blob_storage?
-        blob_storage_client.get_presigned_url("GET", Config.ubicloud_images_bucket_name, "#{image_name}-#{vm_host.arch}-#{version}.raw", 60 * 60).to_s
-      elsif image_name == "ubuntu-noble"
-        arch = (vm_host.arch == "x64") ? "amd64" : "arm64"
-        "https://cloud-images.ubuntu.com/releases/noble/release-#{version}/ubuntu-24.04-server-cloudimg-#{arch}.img"
-      elsif image_name == "ubuntu-jammy"
-        arch = (vm_host.arch == "x64") ? "amd64" : "arm64"
-        "https://cloud-images.ubuntu.com/releases/jammy/release-#{version}/ubuntu-22.04-server-cloudimg-#{arch}.img"
-      elsif image_name == "almalinux-8"
-        fail "Only x64 is supported for almalinux-8" unless vm_host.arch == "x64"
-        "https://repo.almalinux.org/almalinux/8/cloud/x86_64/images/AlmaLinux-8-GenericCloud-#{version}.x86_64.qcow2"
-      elsif image_name == "almalinux-9"
-        arch = (vm_host.arch == "x64") ? "x86_64" : "aarch64"
-        "https://repo.almalinux.org/almalinux/9/cloud/#{arch}/images/AlmaLinux-9-GenericCloud-#{version}.#{arch}.qcow2"
-      else
-        fail "Unknown image name: #{image_name}"
-      end
+    @url ||= frame["custom_url"] || blob_storage_client.get_presigned_url("GET", Config.ubicloud_images_bucket_name, "#{image_name}-#{vm_host.arch}-#{version}.raw", 60 * 60).to_s
```

Comment:
Do we store all images in `.raw` format? The original Ubuntu images are in `.img` format, and AlmaLinux images are in `.qcow2` format. The `.raw` output for some images is 3-4 times larger than the original format.

Reply:
I initially thought that we should converge to a single format in our blob storage, but it would require more effort to update the images. It would also make SHA256 verification logic more complex. Finally, as you noted, .raw output is bigger than the other formats. So I changed it to use original format of the image.